src/xdocs/releasenotes.xml

```diff
@@ -6333,7 +6333,7 @@ used in the MT mode.
             Boolean flag in Javadoc Checks that shows a current Javadoc comment has unclosed HTML tags. Author: Piyush Sharma <a href="https://github.com/checkstyle/checkstyle/issues/3311">#3311</a>
           </li>
           <li>
-            Add columnCharIndex field to LocalizedMessage. Author: Timur Tibeyev <a href="https://github.com/checkstyle/checkstyle/issues/4998">#4998</a>
+            Add columnCharIndex field to Violation. Author: Timur Tibeyev <a href="https://github.com/checkstyle/checkstyle/issues/4998">#4998</a>
```

Comment:
I dont think it makes sense to change release notes, please revert

Reply:
Done.